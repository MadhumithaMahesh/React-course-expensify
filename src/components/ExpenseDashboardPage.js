// import React from 'react';
// import ExpenseList from './ExpenseList'
// const ExpenseDashboardPage = () => (
//     <div>
//       This is from my dashboard components
//       <ExpenseList></ExpenseList>
//     </div>
//   );

//   export default ExpenseDashboardPage

import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilter from './ExpenseListFilter';

const ExpenseDashboardPage = () => (
  <div>
    <ExpenseListFilter/>
    <ExpenseList />
  </div>
);

export default ExpenseDashboardPage;
