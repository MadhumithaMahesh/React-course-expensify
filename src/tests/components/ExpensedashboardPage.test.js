import React from 'react'
import {shallow} from 'enzyme'
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage'


test('should return an object from ExpenseDashboardPage',()=>
{
    const wrapper = shallow(<ExpenseDashboardPage></ExpenseDashboardPage>)
    expect(wrapper).toMatchSnapshot()
})