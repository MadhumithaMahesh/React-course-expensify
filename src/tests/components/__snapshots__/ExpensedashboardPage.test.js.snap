// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return an object from ExpenseDashboardPage 1`] = `
<div>
  <Connect(ExpenseListFilter) />
  <Connect(ExpenseList) />
</div>
`;
